### DISCUSSION TITLE
What is meaning  of pandas ?

### DISCUSSION THREAD

**Discussion by nouman073** [AammarTufail/python_ka_chilla2023#22] [Q&A]
I tried to means of pandas but  different websites show the medical term "Paediatric autoimmune neuropsychiatric disorders associated with streptococcal infections"
anyone knows about it  places writedown in the comments

**Comment by baberibrar** [comment]
> I tried to means of pandas but different websites show the medical term "Paediatric autoimmune neuropsychiatric disorders associated with streptococcal infections" anyone knows about it places writedown in the comments

Pandas stands for “Python Data Analysis Library ”. According to the Wikipedia page on Pandas, “the name is derived from the term “panel data”, an econometrics term for multidimensional structured data sets.” But I think it's just a cute name for a super-useful Python library!
**source**: [https://towardsdatascience.com/a-quick-introduction-to-the-pandas-python-library-f1b678f34673](url)

**Reply by nouman073** [reply]
Thanks
